src/resumes/resumes.controller.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  Controller,
+  HttpStatus,
+  Post,
+  UploadedFile,
+  UseInterceptors,
+} from '@nestjs/common';
+import { ResumesService } from './resumes.service';
+import { ResponseUtil } from '@/shared/utils/response.util';
+import { FileInterceptor } from '@nestjs/platform-express';
+import { CustomHttpException } from '@/shared/exceptions/custom-http-exception';
+
+@Controller()
```

Comment:
as I said this will live under the professionals module

then it need to be routed `professionals/parse-resume`